### PR TITLE
feat: added demo Pulse auto play & sliding effect POC

### PR DESCRIPTION
**Pull request by duyhungtnn**
### Demo
[demo.webm](https://github.com/user-attachments/assets/f37a8843-e4b9-4c58-b175-3228dcdfb50b)

### CODE REVIEW

File: TrinityAudioDemo/PulseSlidingViewController.swift [hunk 1]
```diff
@@ -14,6 +14,10 @@ class PulseSlidingViewController: UIViewController {
     @IBOutlet var triniyPlayerView: TrinityPlayerView!
     @IBOutlet var loadingIndicator: UIActivityIndicatorView!
     @IBOutlet weak var playerIdLb: UILabel!
+    @IBOutlet weak var playerHeightConstraint: NSLayoutConstraint!
+    @IBOutlet weak var playerTopConstraint: NSLayoutConstraint!
+    
+    var autoPlay = false
```
Comment (ilyaa-trinityaudio) [inline TrinityAudioDemo/PulseSlidingViewController.swift]: May be we should set it  to true (like our only consumer will use case)?

**Review by ilyaa-trinityaudio — APPROVED** [review]

File: TrinityAudioDemo/PulseSlidingViewController.swift (hunk 1 above)
Comment (duyhungtnn) [reply TrinityAudioDemo/PulseSlidingViewController.swift]: @ilyaa-trinityaudio 

yes, I set it to be "true" by default https://github.com/TrinityAudioSDK/trinityaudio-ios-demo/pull/11/commits/ba65386e075a085284e0b23bf874a3878d29b7af

**Review by ilyaa-trinityaudio — APPROVED** [review]

**Comment by ilyaa-trinityaudio** [comment]
@duyhungtnn  I can merge this PR?

**Comment by duyhungtnn** [comment]
@ilyaa-trinityaudio yes, its ready

**Pull request merged**